src/xml_document.cc

```diff
@@ -244,17 +246,25 @@ NAN_METHOD(XmlDocument::FromHtml)
     xmlResetLastError();
     xmlSetStructuredErrorFunc(reinterpret_cast<void*>(&errors), XmlSyntaxError::PushToArray);
 
+
+    static const int DEFAULT_PARSING_OPTS = 0;
```

Comment:
wasn't sure of the best practice for placing constants. I can put these in the header file if preferred